action.yml

```
@@ -138,27 +138,34 @@ runs:
+        TAG_NAME="lokalise-sync-${BRANCH_NAME}"
         echo "Looking for tag: $TAG_NAME"
 
-        git fetch --quiet origin "tag:$TAG_NAME" 2>/dev/null || true
-
-        if TAG_SHA=$(git rev-list -n1 "$TAG_NAME" 2>/dev/null); then
-          echo "Found tag '$TAG_NAME': $TAG_SHA"
-          echo "base_sha=$TAG_SHA" >> $GITHUB_OUTPUT
-          exit 0
+        if git ls-remote --exit-code --refs --tags origin "refs/tags/$TAG_NAME" >/dev/null 2>&1; then
+          echo "Tag '$TAG_NAME' found on remote."
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)